### PR TITLE
feat/Home page

### PR DESCRIPTION
**Pull request by darshandamre**
## Home page
[Preview](https://52shop-web-git-home-page-darshandamre.vercel.app/)

### Files to check
 - [ ] src/components/Categories/Categories.js
 - [ ] src/components/Header/Header.js
 - [ ] src/components/NewArrivals/NewArrivals.js
 - [ ] src/pages/Home/Home.js

you can ignore other files, they're lock-files, config files, styles, images

### CODE REVIEW

File: src/components/Categories/Categories.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { categoryList } from "../../utils/categoryList";
```
Comment (TejasShekar) [inline src/components/Categories/Categories.js:3]: Use absolute paths

**Review by TejasShekar — APPROVED** [review]
Code is well structured. Good job.

File: src/components/Categories/Categories.js (hunk 1 above)
Comment (tanishk15sharma) [reply src/components/Categories/Categories.js:3]: +1

**Review by tanishk15sharma — APPROVED** [review]
I liked how you are making all the `images` in equal size , and using proper naming conventions

**Pull request merged**